resources/boilerplates/flattenData.js

```diff
@@ -1,10 +1,11 @@
 export function flattenChildData(element) {
   const flatData ={
     typeId: element.typeId,
+    instances: []
   };
 
-  if (element.instances) {
-    flatData.instances = element.instances.map(
+  if (element.instances && element.instances.length > 0) {
+    flatData.instances = element.instances.filter(child => !!child).map(
```

Comment:
Is it really necessary to check whether children is empty?  Of course the map would return an empty array in such a case.  Also, are the nulls not being filtered out now by the back end?  They really should be I think. If not, that should be a fix.

Reply:
Yeah, allow me to review this.